core/src/trezor/lvglui/scrs/homescreen.py

```
@@ -1255,39 +1262,35 @@ def on_layer2_covers_screen():
-                Layer2Manager.set_animating(False)
-                Layer2Manager.cleanup_timers()
+                l2_manager.set_animating(False)
+                l2_manager.cleanup_timers()
                 gc.collect()  # Reclaim animation callback objects
 
             try:
-                Layer2Manager.with_lvgl_timer_pause(animate_cb, 0, 200)
+                l2_manager.with_lvgl_timer_pause(
+                    animate_cb, LAYER2_VISIBLE_Y, LAYER2_ANIM_TIME_MS
+                )
             except Exception:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)